### ISSUE TITLE
TypeError: Cannot read property 'getDiff' of undefined when calling median()

### ISSUE THREAD

**Issue by mklaber** [alexandrusavin/exectimer#5]
When a timer's `ticks` array has an odd number of ticks (i.e., it has been started and stopped once, thrice, etc.), a call to `median()` fails with:

```
TypeError: Cannot read property 'getDiff' of undefined
    at Object.timers.(anonymous function).median (...exectimer/index.js:29:42)
```

To reproduce: 

```js
var t = require('exectimer');

var timer = new t.Tick('foo');

timer.start();
console.log('started foo');

timer.stop();
console.log('stopped foo');

console.log(t.timers.foo.count()); // works; reports "1"

console.log(t.timers.foo.median()); // fails; reports TypeError: Cannot read property 'getDiff' of undefined at Object.timers.(anonymous function).median
```

However, this works:

```js
var t = require('exectimer');

var timer = new t.Tick('foo');

timer.start();
console.log('started foo');

timer.stop();
console.log('stopped foo');

timer.start();
console.log('started foo again');

timer.stop();
console.log('stopped foo again');

console.log(t.timers.foo.count()); // works; reports "2"

console.log(t.timers.foo.median()); // works
```

**Comment by alexandrusavin** [comment]
Thanks!
I'll look into it as soon as I get some free time.

**Comment by alexandrusavin** [comment]
Fixed in v1.1.0
